test/generator-support.test.ts

```
@@ -0,0 +1,493 @@
+        const fluentGen = fluent
+          .gen()
+          .addClass("processed")
+          .style({ fontWeight: "bold" })
+          .flow();
+
+        for await (const fn of fluentGen) {
+          if (typeof fn === "function") {
+            fn(element);
+          }
+        }
+      };
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)